diff --git a/msvc/config.h b/msvc/config.h
index d4032df..399d013 100644
--- a/msvc/config.h
+++ b/msvc/config.h
@@ -35,16 +35,6 @@
 /* 64bit subdirectory for WinUSB redist files ("x64" or "amd64") */
 #define X64_DIR "x64"
 
-/* embed libusb0 driver files from the following location */
-#ifndef LIBUSB0_DIR
-#define LIBUSB0_DIR "D:/libusb-win32"
-#endif
-
-/* embed libusbK driver files from the following location */
-#ifndef LIBUSBK_DIR
-#define LIBUSBK_DIR "D:/libusbK/bin"
-#endif
-
 /* embed user defined driver files from the following location */
 #ifndef USER_DIR
 // #define USER_DIR "C:/signed-driver"
