diff --git a/libwdi/libwdi.c b/libwdi/libwdi.c
index 56948ac..88c242b 100644
--- a/libwdi/libwdi.c
+++ b/libwdi/libwdi.c
@@ -316,7 +316,7 @@ void GetWindowsVersion(void)
  * Converts a windows error to human readable string
  * uses retval as errorcode, or, if 0, use GetLastError()
  */
-char *windows_error_str(DWORD retval)
+static char *windows_error_str(DWORD retval)
 {
 	static char err_string[STR_BUFFER_SIZE];
 	DWORD size, presize, error_code, format_error;
